Jenkinsfile

```diff
@@ -0,0 +1,153 @@
+#!/usr/bin/env groovy
+
+// can't just use BUILD_TAG because qdr operator limits name of resources to 60 chars
+def namespace = env.JOB_BASE_NAME + '-' + env.BUILD_NUMBER
+namespace = namespace.toLowerCase()
+
+def stages_failed = false;
+
+def stf_resource = """
+apiVersion: infra.watch/v1beta1
+kind: ServiceTelemetry
+metadata:
+  name: default
+  namespace: ${namespace}
+spec:
+  alerting:
+    alertmanager:
+      storage:
+        strategy: ephemeral
+      receivers:
+        snmpTraps:
+          enabled: true
+  backends:
+    events:
+      elasticsearch:
+        enabled: true
+        storage:
+          strategy: ephemeral
+    metrics:
+      prometheus:
+        enabled: true
+        storage:
+          strategy: ephemeral
+  transports:
+    qdr:
+      enabled: true
+      deployment_size: 1
```

Comment:
In the spec you should be using camelCase not snake_case. The snake_case conversion happens at the Operator level.